test/suites/network/multi-nic.robot

```diff
@@ -0,0 +1,186 @@
+*** Settings ***
+Documentation       Tests related to MicroShift running in a multiple NIC environment
+
+Resource            ../../resources/ostree-health.resource
+Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-network.resource
+Resource            ../../resources/kubeconfig.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           network
+
+
+*** Variables ***
+${USHIFT_HOST_IP1}      ${EMPTY}
+${USHIFT_HOST_IP2}      ${EMPTY}
+${NIC1_NAME}            ${EMPTY}
+${NIC2_NAME}            ${EMPTY}
+${NICS_COUNT}           2
+${NMCLI_CMD}            nmcli -f name,type connection | awk '$2 == "ethernet" {print $1}' | sort
+
+
+*** Test Cases ***
+Verify MicroShift Runs On Both NICs
+    [Documentation]    Verify MicroShift can run in the default configuration
+
+    # Wait for MicroShift API readiness and run verification
+    Wait For MicroShift
+    Verify Hello MicroShift LB
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}    ${USHIFT_HOST_IP2}
+
+Verify MicroShift Runs On First NIC
+    [Documentation]    Verify MicroShift can run on the first NIC
+
+    ${cur_pid}=    MicroShift Process ID
+
+    Login Switch To IP1
+    Nmcli Connection Control    down    ${NIC2_NAME}
+
+    # MicroShift should restart due to IP change
+    Wait Until MicroShift Process ID Changes    ${cur_pid}
+    Wait For MicroShift Service
+    Setup Kubeconfig
+
+    # Wait for MicroShift API readiness and run verification
+    Wait For MicroShift
+    Verify Hello MicroShift LB
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}    ${EMPTY}
+
+    # Rebooting MicroShift host restores the network configuration
+    [Teardown]    Run Keywords
+    ...    Reboot MicroShift Host
+    ...    Login Switch To IP1
+    ...    Wait For Healthy System
+
+Verify MicroShift Runs On Second NIC
```

Comment:
This is very similar to `Verify MicroShift Runs On First NIC`. Do you think it's worth the effort to create a generalized keyword or all the vars would only make it harder to understand?